GOVERNANCE.md

```diff
@@ -0,0 +1,462 @@
+# Open Cluster Management Project Governance
+
+## Table of Contents
+
+- [Values](#values)
+- [Project Governance](#project-governance)
+  - [Subproject Categories](#subproject-categories)
+  - [Core Subproject Governance](#core-subproject-governance)
+  - [Default Extension/Addon Governance](#default-extensionaddon-governance)
+  - [Adding New Subprojects](#adding-new-subprojects)
+  - [Removing Projects](#removing-projects)
+- [Contributor Ladder](#contributor-ladder)
+- [Steering Committee](#steering-committee)
+  - [Duties](#duties)
+  - [Members](#members)
+  - [Elections](#elections)
+  - [Decision Making](#decision-making)
+- [Security](#security)
+- [Release Process](#release-process)
+- [Code of Conduct](#code-of-conduct)
+
+# Values
+
+Open Cluster Management and its leadership embrace the following values:
+
+- Openness: Open Cluster Management is an open source community driven project.
+- Distributed design: Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstones of our world-wide community.
+- Fairness: Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+- Diversity and inclusion: Everyone are welcome to participate in the Open Cluster Management project regardless of gender, gender identity, sexual orientation, disability, race, ethnicity, age, religion or economic status.
+
+# Project Governance
+
+Open Cluster Management follows a federated governance model with two distinct categories of subprojects, each with different governance requirements and expectations:
+
+## Subproject Categories
+
+### Core Subprojects
+Core subprojects form the foundation of Open Cluster Management and are maintained by the OCM core team with strict governance requirements:
+
+- [OCM](https://github.com/open-cluster-management-io/ocm) - Core cluster management functionality
+- [API](https://github.com/open-cluster-management-io/api) - Core APIs and CRDs
+- [Clusteradm](https://github.com/open-cluster-management-io/clusteradm) - Command-line tool
+- [Addon Framework](https://github.com/open-cluster-management-io/addon-framework) - Framework for building addons
+- [SDK](https://github.com/open-cluster-management-io/sdk-go) - Go SDK for OCM
+
+**Core Subproject Requirements:**
+- Maintained by OCM core maintainers and approved contributors
+- Follow strict release cycles and version compatibility guarantees
+- Adhere to high code quality standards and comprehensive testing
+- Provide long-term support and stability commitments
+- Must follow the governance rules defined in this document
+
+### Extensions and Addons
+Extensions and addons extend OCM functionality and may be contributed by external users and community contributors:
```

Comment:
external users are not quite appropriate statement. we do not need to mention who is contributing. These would be extension above core, they are optional component to run on ocm-core for different use cases.

Reply:
Got it, the description is updated, PTAL